test/plugin/scenarios/servicecomb-2.x-scenario/support-version.list

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+2.6.0
+2.7.0
```

Comment:
Are 2.0 - 2.5 all not important? @lbc97 @liubao68

Reply:
Our team leader told me to test the two versions.

Reply:
Do you mean someone in the ServiceComb community/PMC team? If so, I am fine.

Reply:
I think is fine. We are going to start from a newer base version and we always recommend users to upgrade to a newer version.